### ISSUE TITLE
IDE0060 false positive for method used as a delegate in nested type

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn#44226]
See https://github.com/dotnet/roslyn/pull/44199/files#diff-346d613909e7c03e101f681834c8e015R22 in `VisualBasicEscapingReducer.vb`

```vb
#Disable Warning IDE0060 ' Remove unused parameter - False positive, used as a delegate in a nested type.
        ' TODO: File a bug for this 'IDE0060' false positive.
        Private Shared Function TryUnescapeToken(identifier As SyntaxToken, semanticModel As SemanticModel, optionSet As OptionSet, cancellationToken As CancellationToken) As SyntaxToken
#Enable Warning IDE0060 ' Remove unused parameter
            If Not identifier.IsBracketed Then
                Return identifier
            End If
```
